src/components/forms/CalendarDemo.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import {FormField} from './forms';
+import {Styles} from '../../libs/theme';
+import {FormFieldTypes, FormValidators} from './forms';
+import dayjs from 'dayjs';
+
+export const CalendarDemo = () => {
```

Comment:
Why is this named as a "Demo"?

Reply:
It doesn't look like this component is used, is this more for demonstration purposes?

Reply:
@snf2ye Good catch!  I accidentally committed this by mistake.  I'll remove this file.